custom-eureka-client/README.md

```diff
@@ -0,0 +1,56 @@
+# Access Azure Spring Cloud managed Spring Cloud Service Registry
+
+This sample shows how to access Azure Spring Cloud managed Spring Cloud Service Registry through Azure RBAC when your applications are running outside Azure Spring Cloud.
+
+## Prerequisite
+
+* [JDK 8](https://docs.microsoft.com/en-us/azure/java/jdk/java-jdk-install)
+* [Maven 3.0 and above](http://maven.apache.org/install.html)
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) or [Azure Cloud Shell](https://docs.microsoft.com/en-us/azure/cloud-shell/overview)
+
+## How to run
+
+1. Clone this repo and go to folder
+
+    ```bash
+    git clone https://github.com/Azure-Samples/Azure-Spring-Cloud-Samples
+    cd Azure-Spring-Cloud-Samples/custom-eureka-client
+    ```
+
+1. Install Azure CLI extension for Azure Spring Cloud
+
+    ```bash
+    az extension add --name spring-cloud
+    ```
+
+1. Create an instance of Azure Spring Cloud
+
+    ```bash
+    az spring-cloud create -n <service name> -g <resource group name>
+    ```
+
+1. Create a service principal to access the Config Server in your application.
+    ```bash
+    az ad sp create-for-rbac --scopes /subscriptions/<subscription id>/resourceGroups/<resource group name>/providers/Microsoft.AppPlatform/Spring/<service name> --role "Azure Spring Cloud Service Registry Contributor"
+    ```
+   The output includes credentials you will need in the next step.
+   
+1. Update the `application.properties` under `src/main/resources` with the credentials you get in the above step and your Azure Spring Cloud instance.

```

Comment:
Are we going to give some brief introduction about the required code changes to customers?

Reply:
The explanation is included in the Azure Spring Cloud doc. https://github.com/MicrosoftDocs/azure-docs-pr/pull/169498
See section `## Register Spring Boot apps to Spring Cloud Config Server and Service Registry managed by Azure Spring Cloud`
Is it expected to put here as well?

Reply:
I see. That's okay then.
My personal preference would be making this code sample self-contained. But we can leave it as now first. If there is any feedback later, we can add the explanation.